packages/discovery-provider/ddl/migrations/0114_country_list.sql

```diff
@@ -0,0 +1,308 @@
+('SK', 'SLOVAKIA', 'Slovakia', 'SVK', 703, 421),
+('SL', 'SIERRA LEONE', 'Sierra Leone', 'SLE', 694, 232),
+('SM', 'SAN MARINO', 'San Marino', 'SMR', 674, 378),
+('SN', 'SENEGAL', 'Senegal', 'SEN', 686, 221),
+('SO', 'SOMALIA', 'Somalia', 'SOM', 706, 252),
+('SR', 'SURINAME', 'Suriname', 'SUR', 740, 597),
+('SS', 'SOUTH SUDAN', 'South Sudan', 'SSD', '728', '211'),
+('ST', 'SAO TOME AND PRINCIPE', 'Sao Tome and Principe', 'STP', 678, 239),
+('SV', 'EL SALVADOR', 'El Salvador', 'SLV', 222, 503),
+('SX', 'SINT MAARTEN', 'Sint Maarten', 'SXM', '534', '1'),
+('SY', 'SYRIAN ARAB REPUBLIC', 'Syrian Arab Republic', 'SYR', 760, 963),
+('SZ', 'SWAZILAND', 'Swaziland', 'SWZ', 748, 268),
+('TC', 'TURKS AND CAICOS ISLANDS', 'Turks and Caicos Islands', 'TCA', 796, 1649),
+('TD', 'CHAD', 'Chad', 'TCD', 148, 235),
+('TF', 'FRENCH SOUTHERN TERRITORIES', 'French Southern Territories', 'ATF', 260, 0),
+('TG', 'TOGO', 'Togo', 'TGO', 768, 228),
+('TH', 'THAILAND', 'Thailand', 'THA', 764, 66),
+('TJ', 'TAJIKISTAN', 'Tajikistan', 'TJK', 762, 992),
+('TK', 'TOKELAU', 'Tokelau', 'TKL', 772, 690),
+('TL', 'TIMOR-LESTE', 'Timor-Leste', 'TLS', 626, 670),
+('TM', 'TURKMENISTAN', 'Turkmenistan', 'TKM', 795, 993),
+('TN', 'TUNISIA', 'Tunisia', 'TUN', 788, 216),
+('TO', 'TONGA', 'Tonga', 'TON', 776, 676),
+('TR', 'TÜRKIYE', 'Türkiye, The Republic of', 'TUR', 792, 90),
+('TT', 'TRINIDAD AND TOBAGO', 'Trinidad and Tobago', 'TTO', 780, 1868),
+('TV', 'TUVALU', 'Tuvalu', 'TUV', 798, 688),
+('TW', 'TAIWAN, PROVINCE OF CHINA', 'Taiwan, Province of China', 'TWN', 158, 886),
+('TZ', 'TANZANIA, UNITED REPUBLIC OF', 'Tanzania, United Republic of', 'TZA', 834, 255),
+('UA', 'UKRAINE', 'Ukraine', 'UKR', 804, 380),
+('UG', 'UGANDA', 'Uganda', 'UGA', 800, 256),
+('UM', 'UNITED STATES MINOR OUTLYING ISLANDS', 'United States Minor Outlying Islands', 'UMI', 581, 1),
+('US', 'UNITED STATES', 'United States', 'USA', 840, 1),
+('UY', 'URUGUAY', 'Uruguay', 'URY', 858, 598),
+('UZ', 'UZBEKISTAN', 'Uzbekistan', 'UZB', 860, 998),
+('VA', 'HOLY SEE (VATICAN CITY STATE)', 'Holy See (Vatican City State)', 'VAT', 336, 39),
+('VC', 'SAINT VINCENT AND THE GRENADINES', 'Saint Vincent and the Grenadines', 'VCT', 670, 1784),
+('VE', 'VENEZUELA', 'Venezuela', 'VEN', 862, 58),
+('VG', 'VIRGIN ISLANDS, BRITISH', 'Virgin Islands, British', 'VGB', 92, 1284),
+('VI', 'VIRGIN ISLANDS, U.S.', 'Virgin Islands, U.s.', 'VIR', 850, 1340),
+('VN', 'VIET NAM', 'Vietnam', 'VNM', 704, 84),
+('VU', 'VANUATU', 'Vanuatu', 'VUT', 548, 678),
+('WF', 'WALLIS AND FUTUNA', 'Wallis and Futuna', 'WLF', 876, 681),
+('WS', 'SAMOA', 'Samoa', 'WSM', 882, 684),
+('XK', 'KOSOVO', 'Kosovo', 'XKX', 0, 383),
+('YE', 'YEMEN', 'Yemen', 'YEM', 887, 967),
+('YT', 'MAYOTTE', 'Mayotte', 'MYT', 175, 269),
+('ZA', 'SOUTH AFRICA', 'South Africa', 'ZAF', 710, 27),
+('ZM', 'ZAMBIA', 'Zambia', 'ZMB', 894, 260),
+('ZW', 'ZIMBABWE', 'Zimbabwe', 'ZWE', 716, 263)
+
+-- ('WW', 'WORLD WIDE', 'World Wide', 'WWW', -1, -1)
```

Comment:
what's the thought on WW?

Reply:
I kept it out of table for now since it's non-standard.  
We already have a mix of `null` and empty string for missing country... I didn't want to add a potential 3rd missing value into the mix on the database side.

Report sql adds it in as a country code with `UNION (null, 'WW')` and rewrites empty string to null for existing data.